config/config/src/main/java/io/helidon/config/ProviderImpl.java

```diff
@@ -116,6 +117,10 @@ public synchronized Config last() {
         return lastConfig;
     }
 
+    synchronized Optional<ConfigNode> lazyValue(String string) {
```

Comment:
Synchronized?

Reply:
Fixed. Also fixed usage of `ConcurrentHashMap.computeIfAbsent` for possibly blocking operation.